utils/progressbar/progressbarmng.go

```diff
@@ -194,11 +194,25 @@ func (bm *ProgressBarMng) NewStringProgressBar(headline string, updateFn func()
 		mpb.BarRemoveOnComplete(),
 		mpb.PrependDecorators(
 			decor.Name(headline),
-			decor.Any(func(statistics decor.Statistics) string {
-				return updateFn()
+		),
+	)
+	return pb
+}
+
+func (bm *ProgressBarMng) NewDoubleValuesProgressBar(headLine string, color Color, totalTasks int64, updateFunc func() string) *TasksProgressBar {
```

Comment:
Let's add a clear description for this function, expalining what `DoubleValueProgressBar` actually is.
Let's do the same for the `NewStringProgressBar`, `NewCounterProgressBar`, `NewDoneTasksProgressBar` and `NewTasksProgressBar` functions. This should really help the reader understand the differences between them.

Reply:
Done!